net2cog/utilities.py

```diff
@@ -21,6 +21,18 @@
     'float32',
     'float64',
 ]
+DIM_STANDARD_NAME = [
+    'projection_x_coordinate',
+    'projection_y_coordinate',
+    'longitude',
+    'latitude',
+]
+DIM_UNITS = [
+    'degrees_east',
+    'degrees_west',
```

Comment:
Do you have an example of `degrees_west` in the wild? It's not a CF-Convention compliant attribute value for longitude `units`.